get-started/SigOpt_Experiment_And_Optimization_Demo.ipynb

```diff
@@ -0,0 +1 @@
ax': 1.0}\n","        },\n","        {\n","            'name': 'min_split_loss',\n","            'type': 'double',\n","            'bounds': {'min': 0.0, 'max': 10.0}\n","        }\n","    ],\n","    'observation_budget': 20\n","}"],"execution_count":null,"outputs":[]},{"cell_type":"markdown","metadata":{"id":"gxcokWV1-7jZ"},"source":["SigOpt will conveniently output the Experiment link in the terminal so you can check your experiment was created."]},{"cell_type":"markdown","metadata":{"id":"X8imk4IQlWlx"},"source":["## Execute SigOpt Optimization \n","Let's run our optimization using the `%%optimize` magic command. SigOpt will pick up the `experiment` configuration automatically  and conveniently output links in the terminal to the surrent training run on our web application."]},{"cell_type":"code","metadata":{"id":"ktWWkiQxoaix"},"source":["%%optimize My_First_Optimization\n","run_and_track_in_sigopt()"],"execution_count":null,"outputs":[]},{"cell_type":"markdown","metadata":{"id":"gq2TkUqBz_aP"},"source":["## Visualize Results\n","\n","You can click on any of the run links above and view your completed run in our web application. Here's a view of a training run page:\n","\n","<img src=\"https://media.giphy.com/media/dofe86JH6nbPVbJyzy/giphy.gif\" width=\"600\"/>\n","\n","The charts on the training run page show how it compares on key metrics with other runs in the same project.\n","\n","From the Run page, click on the Project Name at the top of the page to navigate to your project. At the project level, you can compare training runs, sort and filter through your training runs and view useful charts to gain insight into everything you've tried.\n","\n","<img src=\"https://media.giphy.com/media/GY3CyrLTecBD9oCtce/giphy.gif\" width=\"600\"/>\n","\n","From the Project page, click on the Experiments tab, and click on the experiment you just created. The experiment Summary page features the experiment best value and shows experiment improvement in a grapth that plots the best observed model metric throughout the course of your experiment.\n","\n","The experiment Analyis page features additional visualizations to help you gain insight into your optimization problem, including Paramater Importance, Parallel Coordinates, and interactive graphs that help you create 2D and 3D representation of your metric and parameter space."]},{"cell_type":"markdown","metadata":{"id":"Pd67ldz8vTYA"},"source":["## From Experiments To Training Runs\n","\n","In this demo we've covered the recommended way to instrument and optimize your model, and visualize your results with SigOpt. You learned that experiments are collections of runs that search through a defined parameter space for one or more metrics. Check out this ([notebook](https://colab.research.google.com/github/sigopt/sigopt-examples/blob/master/get-started/sigopt_training_run_demo.ipynb/)) for a closer look at a single training run, and see how to track one-off training runs without creating an experiment. "]}]}
```

Comment:
We would never edit this directly as text, but as a notebook itself, right? If the latter, then single-line is fine; but if it's not hard to reformat as the multi-line (like the below notebook), then it might be nice for understanding tweaks we make over time.

Not blocking, but if it's simple-enough, let's switch the format.

Reply:
The takeaway is that downloading the file from Drive is a single string but downloading the .ipynb file from the google colab notebook itself comes with the nice formatting.

Reply:
Ohh good to know as we add more of these